### PR TITLE
Parties endpoint

### PR DESCRIPTION
**Pull request by lekshmimallika-aot**
*Issue #:* /bcgov/entity#10415

*Description of changes:*
Added endpoint for parties
Added tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

**Review by vysakh-menon-aot — APPROVED** [review]

File: legal-api/src/legal_api/resources/v2/business/business_parties.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+# limitations under the License.
+"""Retrieve the parties for the entity."""
+from datetime import datetime
+from http import HTTPStatus
+
+from flask import jsonify, request
+from flask_cors import cross_origin
+
+from legal_api.models import Business, PartyRole
+
```
Comment (thorwolpert) [inline legal-api/src/legal_api/resources/v2/business/business_parties.py:22]: Needs the JWT
```suggestion
from legal_api.utils.auth import jwt
```

File: legal-api/src/legal_api/resources/v2/business/business_parties.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+from .bp import bp
+
+
+@bp.route('/<string:identifier>/parties', methods=['GET', 'OPTIONS'])
+@bp.route('/<string:identifier>/parties/<int:party_id>', methods=['GET', 'OPTIONS'])
+@cross_origin(origin='*')
+def get_parties(identifier, party_id=None):
```
Comment (thorwolpert) [inline legal-api/src/legal_api/resources/v2/business/business_parties.py:29]: Needs at least a valid JWT
```suggestion
@jwt.requires_auth
def get_parties(identifier, party_id=None):
```

**Review by thorwolpert — COMMENTED** [review]
LGTM, but it should be a protected end-point

File: legal-api/src/legal_api/resources/v2/business/business_parties.py (hunk 2 above)
Comment (lekshmimallika-aot) [reply legal-api/src/legal_api/resources/v2/business/business_parties.py:29]: Sure. I totally missed that

**Pull request merged**